src/vcSettings.cpp

```diff
@@ -250,14 +250,48 @@ bool vcSettings_Load(vcSettings *pSettings, bool forceReset /*= false*/, vcSetti
     pSettings->maptiles.mapEnabled = data.Get("maptiles.enabled").AsBool(true);
     pSettings->maptiles.demEnabled = data.Get("maptiles.demEnabled").AsBool(true);
 
-    udStrcpy(pSettings->maptiles.mapType, data.Get("maptiles.type").AsString("euc-az-aerial"));
+    pSettings->maptiles.activeLayerCount = data.Get("maptiles.layerCount").AsInt(0);
```

Comment:
`maptiles.layerCount` isn't really necessary, you can do this with `data.Get("maptiles.layer").ArrayLength()`?